lighthouse-core/gather/drivers/driver.js

```diff
@@ -213,6 +221,76 @@ class Driver {
     });
   }
 
+  getServiceWorkerRegistrations() {
+    return new Promise(resolve => {
+      this.once('ServiceWorker.workerRegistrationUpdated', data => {
+        this.sendCommand('ServiceWorker.disable');
+        resolve(data);
+      });
+
+      this.sendCommand('ServiceWorker.enable');
+    });
+  }
+
+  /**
+   * Checks all serviceworkes and see if any duplications are running
+   * @param {!string} pageUrl
+   * @return {!Promise}
+   */
+  checkForMultipleTabsAttached(pageUrl) {
+    // Get necessary information of serviceWorkers
+    const getRegistrations = this.getServiceWorkerRegistrations();
+    const getVersions = this.getServiceWorkerVersions();
+    const getActiveTabId = this.getCurrentTabId();
+
+    return Promise.all([getRegistrations, getVersions, getActiveTabId]).then(res => {
+      const registrations = res[0].registrations;
+      const versions = res[1].versions;
+      const activeTabId = res[2];
+      const parsedURL = parseURL(pageUrl);
+      const origin = `${parsedURL.protocol}//${parsedURL.hostname}` +
+      (parsedURL.port ? `:${parsedURL.port}` : '');
+
+      let swHasMoreThanOneClient = false;
+      registrations
+        .filter(reg => !reg.isDeleted)
```

Comment:
nuke this isDeleted filter. Even if it's deleted (aka unregistered) it came come back from the dead.